presidio-anonymizer/README.MD

```diff
@@ -1 +1,121 @@
-# TODO add the document Omri wrote.
\ No newline at end of file
+# Presidio anonymizer
+
+## Description
+
+The Presidio anonymizer is a Python based service for anonymizing detected PII text
+entities with desired values.
+
+Presidio anonymizer comes by default with the following anonymizers:
+
+- Replace - replaces PII with desired value
+- Redact - removes PII completely from text
+- Hash - hash the PII using sha256
+- FPE - using ff1 algorithm for formatting-Preserving Encryption on the PII
+- Mask - mask the selected amount of characters of the PII with a given character
+
+Since text might have several PIIs, the anonymizer handle those scenarios in the
+following ways:
+
+- One PII - uses a given or default transformation to anonymize and replace the PII text
+  entity.
+- Full overlap of PIIs - When one text have several PIIs, the PII with the higher score
+  will be taken. Between PIIs with similar scores, the selection will be arbitrary.
+- One PII is contained in another - anonymizer will use the PII with larger text.
+- Partial intersection - both will be returned concatenated.
+
+## Installation
+
+### As service:
+
+Run presidio_anonymizer/app.py or docker container and use endpoint:
```

Comment:
this should either be more specific or link to another doc which is.

Reply:
I think it will be more specific once the dockerization of the package will be done WDYT?

Reply:
so maybe write how to run it with flask which is doable at this point

Reply:
Done